Framework/API/inc/MantidAPI/IMDHistoWorkspace.h

```diff
@@ -44,6 +44,10 @@ class DLLExport IMDHistoWorkspace : public IMDWorkspace,
   IMDHistoWorkspace_uptr clone() const {
     return IMDHistoWorkspace_uptr(doClone());
```

Comment:
I think this was discussed on a review a long time ago. Minor point - since the covariance is unnecessary. Maybe just `make_unique` inplace and drop the `doClone` method?